purchases/src/main/kotlin/com/revenuecat/purchases/models/TestStoreProduct.kt

```diff
@@ -0,0 +1,95 @@
+package com.revenuecat.purchases.models
+
+import com.revenuecat.purchases.ProductType
+
+/**
+ * A test-only [StoreProduct] implementation.
+ * This can be used to create mock data for tests or Jetpack Compose previews.
+ */
+data class TestStoreProduct(
```

Comment:
I think we can probably remove some implementations of `StoreProduct` from the tests with this, but we can leave that for another PR.